File: schrottverwaltung/src/main/java/de/fhdortmund/schrottverwaltung/immobilie/Koordinaten.java
package de.fhdortmund.schrottverwaltung.immobilie;

import jakarta.persistence.*;
import lombok.*;

@Entity
@Table(name = "koordinaten")
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Koordinaten {

    private Double xKoordinate;

    private Double yKoordinate;

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
}

File: schrottverwaltung/src/main/kotlin/de/fhdortmund/schrottverwaltung/immoblilie/Koordinaten.kt
package de.fhdortmund.schrottverwaltung.immoblilie

import jakarta.persistence.*

@Entity
@Table(name = "koordinaten")
data class Koordinaten(
    var xKoordinate: Double,
    var yKoordinate: Double,
) {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Long? = null
}